src/main/java/com/wanderersoftherift/wotr/datagen/WotrModifierProvider.java

```diff
@@ -165,79 +185,93 @@ public static void bootstrapModifiers(BootstrapContext<Modifier> context) {
+                0x6495ED) // Other Blueish
         );
-        registerModifier(context, getResourceKey("movement_speed"),
-                new Modifier(generateEqualRollSpread(3,
+        registerModifier(context, getResourceKey("movement_speed"), new Modifier(
+                generateEqualRollSpread(3,
                         List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
                                 attributeModifierEffectGetter(WanderersOfTheRift.id("movement_speed"),
-                                        Attributes.MOVEMENT_SPEED, AttributeModifier.Operation.ADD_VALUE)))))
+                                        Attributes.MOVEMENT_SPEED, AttributeModifier.Operation.ADD_VALUE)))),
+                0x00FA9A) // Greenish
         );
-        registerModifier(context, getResourceKey("projectile_count"),
-                new Modifier(generateEqualRollSpread(2, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
+        registerModifier(context, getResourceKey("projectile_count"), new Modifier(
+                generateEqualRollSpread(2, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
                         attributeModifierEffectGetter(WanderersOfTheRift.id("projectile_count"),
-                                WotrAttributes.PROJECTILE_SPEED, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))))
+                                WotrAttributes.PROJECTILE_SPEED, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))),
+                0xF0E68C) // Yellow-Brownish
         );
-        registerModifier(context, getResourceKey("projectile_pierce"),
-                new Modifier(generateEqualRollSpread(2, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
+        registerModifier(context, getResourceKey("projectile_pierce"), new Modifier(
+                generateEqualRollSpread(2, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
                         attributeModifierEffectGetter(WanderersOfTheRift.id("projectile_pierce"),
-                                WotrAttributes.PROJECTILE_PIERCE, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))))
+                                WotrAttributes.PROJECTILE_PIERCE, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))),
+                0xDAA520) // Goldish
         );
-        registerModifier(context, getResourceKey("projectile_speed"),
-                new Modifier(generateEqualRollSpread(5, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
+        registerModifier(context, getResourceKey("projectile_speed"), new Modifier(
+                generateEqualRollSpread(5, List.of(new ToBeTieredModifierEffect(0.01F, 0.1F,
                         attributeModifierEffectGetter(WanderersOfTheRift.id("projectile_speed"),
-                                WotrAttributes.PROJECTILE_SPEED, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))))
+                                WotrAttributes.PROJECTILE_SPEED, AttributeModifier.Operation.ADD_MULTIPLIED_BASE)))),
+                0x7FFFD4) // Another Blue
```

Comment:
I would probably pull all these constants out (possibly to ColorUtil) and name them.